src/accordion/AccordionState.ts

```diff
@@ -5,33 +5,32 @@ import {
   CompositeInitialState,
 } from "reakit";
 import * as React from "react";
-import {
-  SealedInitialState,
-  useSealedState,
-} from "reakit-utils/useSealedState";
+import { useControllableState } from "@chakra-ui/hooks";
 
 export type AccordionState = CompositeState & {
```

Comment:
let's not self reference the types for the accordion actions. just to reduce confusion. 

```diff
- registerPanel: AccordionActions["registerItem"];
+ registerPanel: CompositeActions["registerItem"];
```

```ts

export type AccordionActions = CompositeActions & {
  /**
   * Moves into and selects an accordion by its `id`.
   */
  select: CompositeActions["move"];
  /**
   * Moves into and unSelects an accordion by its `id` if it's already selected.
   */
  unSelect: CompositeActions["move"];
  /**
   * Sets `selectedId`.
   */
  setSelectedId: React.Dispatch<React.SetStateAction<string | null>>;
  /**
   * Sets `selectedIds`.
   */
  setSelectedIds: React.Dispatch<React.SetStateAction<(string | null)[]>>;
  /**
   * Registers a accordion panel.
   */
  registerPanel: CompositeActions["registerItem"];
  /**
   * Unregisters a accordion panel.
   */
  unregisterPanel: CompositeActions["unregisterItem"];
};
``